specs/wp-post-editor-spec.js

```diff
@@ -863,7 +863,6 @@ test.describe( `[${host}] Editor: Posts (${screenSize})`, function() {
 						assert.equal( errorShown, false, 'There is an error shown on the editor page!' );
 					} );
 					this.postEditorToolbarComponent = new PostEditorToolbarComponent( driver );
-					this.postEditorToolbarComponent.ensureSaved();
```

Comment:
Why does this work not calling this now? The reason we have ensureSaved is to avoid the "unsaved changes" dialogs which sometimes appear as a race condition - are we sure these won't show?

Reply:
It works because it really wasn't needed. We're not trying to change the page or anything, We are clicking the update button which saves it for us.